src/services/actionValidator/documentActionValidator.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { validateDocumentAction } from "./documentActionValidator";
+
+describe("validateDocumentAction", () => {
+  it("should not throw for valid uri", () => {
+    expect.assertions(1);
+    const input = {
+      uri: "https://test.com/doc/123"
+    };
+    expect(() => validateDocumentAction(input)).not.toThrow();
+  });
```

Comment:
Is it useful to add an assertion for the return value of `validateDocumentAction`?